src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -20,14 +20,36 @@ internal class LiveLogger : ILogger
         public LoggerVerbosity Verbosity { get; set; }
         public string Parameters { get; set; }
 
+        /// <summary>
+        /// List of events the logger needs as parameters to the <see cref="ConfigurableForwardingLogger"/>.
+        /// </summary>
+        /// <remarks>
+        /// If LiveLogger runs as a distributed logger, MSBuild out-of-proc nodes might filter the events that will go to the main node using an instance of <see cref="ConfigurableForwardingLogger"/> with the following parameters.
+        /// </remarks>
+        public static readonly string[] ConfigurableForwardingLoggerParameters =
+        {
+            "BUILDSTARTEDEVENT",
+            "PROJECTSTARTEDEVENT",
+            "TARGETSTARTEDEVENT",
+            "TASKSTARTEDEVENT",
+            "BUILDFINISHEDEVENT",
+            "PROJECTFINISHEDEVENT",
+            "TARGETFINISHEDEVENT",
+            "HIGHMESSAGEEVENT",
```

Comment:
Do we really need information about messages to be showed in LiveLogger? I think that errors and warnings are, but I am unsure about messages of different importance.

Reply:
We need _some_ messages at different levels, to extract the right info we need. Right now that looks like just HIGH so we should be able to get some perf out of dropping normal and low.

Eventually I suspect we'll want some info from low-importance messages in some cases. That might make us move away from the CentralForwardingLogger to a custom filtering implementation.

Reply:
Yes, I see, non-high priority messages are dropped. I really hope that we will not need non-high prio messages for a logger to work.

Reply:
Dropping medium and low priority messages indeed helped perf.

Reply:
#8463 will likely help further.

Reply:
I will take a look, thanks!